src/main/java/pro/trevor/tankgame/rule/impl/util/RandomManager.java

```diff
@@ -0,0 +1,7 @@
+package pro.trevor.tankgame.rule.impl.util;
+
+import java.util.Random;
+
```

Comment:
This should be a part of the state. Any changes should probably be deferred to another issue.